jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -173,6 +174,22 @@ public JibContainerBuilder createJibContainerBuilder(
 
       FileCollection snapshotDependencies =
           nonProjectDependencies.filter(file -> file.getName().contains("SNAPSHOT"));
+
+      FileCollection changingDependencies =
+          project.files(
+              project
+                  .getConfigurations()
+                  .getByName(JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME)
+                  .getDependencies()
```

Comment:
I see these lines are duplicated above, so I'd like to see this part factored out as a variable. Like

```java
      Configuration runtimeClasspathConfiguration =
          project.getConfigurations().getByName(JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME);

      FileCollection projectDependencies =
          project.files(
              runtimeClasspathConfiguration
                  .getResolvedConfiguration()
      ...

      FileCollection changingDependencies =
          project.files(
              runtimeClasspathConfiguration
                  .getDependencies()
```